bin/kano-uixinit

```diff
@@ -79,6 +102,8 @@ set_keyboard $kano_keyboard
 # Jump to Dashboard Mode, unless instructed otherwise
 if [ "${DESKTOP_MODE}" != "1" ]; then
     logger --id --tag "info" "kano-uixinit terminates into Dashboard Supervisor"
+    # only run updater once each boot
+    [ "$DESKTOP_RUN" -eq 1 ]  &&  sudo kano-updater ui boot-window
```

Comment:
IIRC, kano-hourglass-app waits for a window. There won't normally be a window, what will kdesk-hourglass-app do in that case @skarbat ?